adapter.go

```diff
@@ -83,49 +103,51 @@ func NewFilteredAdapter(url string) persist.FilteredAdapter {
 }
 
 func (a *adapter) open() {
```

Comment:
Add return `error` better.

Reply:
Great suggestion, I think we should make the error handling of the whole package consistent, the function should be like this
`func NewAdapter (url string, timeout ... interface {}) (persist.Adapter, err)`
All errors should be left to the user to decide

If not, then `panic()` is enough at the time of creation, otherwise it will be processed in panic in other places as well